src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -247,24 +252,27 @@ def excel2resources(
     return resources, True
 
 
-def _prepare_classes_df(resource_dfs: dict[str, pd.DataFrame]) -> tuple[pd.DataFrame, dict[str, pd.DataFrame]]:
-    resource_dfs = {k.strip(): v for k, v in resource_dfs.items()}
+def _do_classes_excel_sheet_compliance(resource_dfs: dict[str, pd.DataFrame]) -> list[Problem]:
     sheet_name_list = list(resource_dfs)
+    problems: list[Problem] = []
+    cls_sheet_name = _find_all_classes(sheet_name_list)
+    if not cls_sheet_name:
+        problems.append(MissingSheetProblem(missing_sheets=["classes"], existing_sheets=sheet_name_list))
+    elif len(cls_sheet_name) > 1:
+        problems.append(ResourceClassSheet(cls_sheet_name))
+    return problems
+
+
+def _find_all_classes(sheet_name_list: list[str]) -> list[str]:
```

Comment:
does this find all class sheets? Then it should be renamed to "_find_all_class_sheet_names"